<ui version="4.0" >
 <class>OBConverterWidget</class>
 <widget class="QWidget" name="OBConverterWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Output Format:</string>
     </property>
     <property name="buddy" >
      <cstring>InputTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Input Format:</string>
     </property>
     <property name="buddy" >
      <cstring>InputTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5" >
    <widget class="QComboBox" name="OutputTypeComboBox" />
   </item>
   <item row="1" column="1" colspan="5" >
    <widget class="QComboBox" name="InputTypeComboBox" />
   </item>
   <item row="3" column="5" >
    <widget class="KPushButton" name="convertFileButton" >
     <property name="text" >
      <string>convert</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="KPushButton" name="selectAllFileButton" >
     <property name="text" >
      <string>select all</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="KPushButton" name="deleteFileButton" >
     <property name="text" >
      <string>delete</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="KPushButton" name="addFileButton" >
     <property name="text" >
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6" >
    <widget class="QListWidget" name="FileListView" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
